### DISCUSSION TITLE
AG Plugin 1.2.0 (BETA 1) crash

### DISCUSSION THREAD

**Discussion by jaacMusic** [apohl79/audiogridder#502] [Help]
Hello Andreas,
I have been trying the new version 1.2.0 (BETA 1) but I haven't been able to load the AG plugin it freezes my DAW when adding it to a track (Cakewalk).
I have tried with both the AG server offline and online but I still get the same reaction it freezes the DAW. I don't have a way to collect logs as I never get the chance to enable it for logging.
![image](https://user-images.githubusercontent.com/68957086/115429319-4bfc6100-a1d1-11eb-9939-7e0914fe4b2a.png)

**Comment by CaBuffalo** [comment]
Are you on Windows? If so, please uninstall 1.1.1 and reinstall 1.2 Beta 1.

On Tue, Apr 20, 2021 at 9:10 AM jaacMusic ***@***.***> wrote:

> Hello Andreas,
> I have been trying the new version 1.2.0 (BETA 1) but I haven't been able
> to load the AG plugin it freezes my DAW when adding it to a track
> (Cakewalk).
> I have tried with both the AG server offline and online but I still get
> the same reaction it freezes the DAW. I don't have a way to collect logs as
> I never get the chance to enable it for logging.
> [image: image]
> <https://user-images.githubusercontent.com/68957086/115429319-4bfc6100-a1d1-11eb-9939-7e0914fe4b2a.png>
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/502>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6DJFOSZ4U6PKIBAL7LTJWRN7ANCNFSM43IPH74A>
> .
>


-- 
Kevin Ma
Supported by Gmail.

**Reply by jaacMusic** [reply]
Yes, for both AG server and DAW. Sorry, I forgot to add that important item.

**Comment by apohl79** [comment]
Can you manually enable logging by setting

`"Logger": true`

in the plugin config (https://audiogridder.com/plugin/#config)?

I tested the hanging problem with cakewalk and it was only caused by the plugin not finding the tray app (which can be caused by not uninstalling a prev. plugin version).

If you see the message "no tray app available" in the plugin log, thats the problem. If not, please share the log.

**Reply by jaacMusic** [reply]
Uninstalling everything and deleting all directories entries for the AG server/Plugin fixed the issues. Thanks!
